src/app/demo/demo.component.ts

```diff
@@ -1,16 +1,23 @@
-import { Component, OnInit } from '@angular/core';
-import {Router} from '@angular/router';
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { Router } from '@angular/router';
+
+type langDir = 'ltr' | 'rtl';
```

Comment:
I would appreciate a comment here that mentions that `ltr` means left to right and `rtl` right to left

Reply:
done.